### PR TITLE
Added staff chat

### PR DESCRIPTION
**Pull request by StudentAlleg**
/staffchat, similar to /clanchat or /allychat

### CODE REVIEW

File: clans/src/main/java/me/mykindos/betterpvp/clans/clans/commands/chatcommands/AllyChatCommand.java [hunk 1]
```diff
@@ -42,13 +42,14 @@ public void execute(Player player, Client client, String... args) {
         if(gamerOptional.isPresent()) {
             boolean allyChatEnabled = true;
             Gamer gamer = gamerOptional.get();
-            Optional<Boolean> clanChatEnabledOptional = gamer.getProperty(GamerProperty.ALLY_CHAT);
-            if(clanChatEnabledOptional.isPresent()){
-                allyChatEnabled = !clanChatEnabledOptional.get();
+            Optional<Boolean> staffChatEnabledOptional = gamer.getProperty(GamerProperty.ALLY_CHAT);
```
Comment (Mykindos) [inline clans/src/main/java/me/mykindos/betterpvp/clans/clans/commands/chatcommands/AllyChatCommand.java:45]: you are still checking ally chat here 🤔

Comment (StudentAlleg) [reply clans/src/main/java/me/mykindos/betterpvp/clans/clans/commands/chatcommands/AllyChatCommand.java]: fixed

File: core/src/main/java/me/mykindos/betterpvp/core/gamer/GamerManager.java [hunk 2]
```diff
@@ -31,6 +34,10 @@ public Optional<Gamer> getGamerByName(String name) {
         return objects.values().stream().filter(gamer -> gamer.getClient().getName().equalsIgnoreCase(name)).findFirst();
     }
 
+    public List<Gamer> getOnlineOfRank(Rank rank) {
```
Comment (Mykindos) [inline core/src/main/java/me/mykindos/betterpvp/core/gamer/GamerManager.java:37]: This isn't really returning online gamers, perhaps just called it getGamersWithRank

**Review by Mykindos — CHANGES_REQUESTED** [review]

File: core/src/main/java/me/mykindos/betterpvp/core/gamer/GamerManager.java (hunk 2 above)
Comment (StudentAlleg) [reply core/src/main/java/me/mykindos/betterpvp/core/gamer/GamerManager.java]: Yeah I thought so, fixed.

**Review by Mykindos — APPROVED** [review]

**Pull request merged**